src/Migrator.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace BuildEmpire\Apartment;
+
+use Illuminate\Support\Str;
+
+class Migrator
+{
+    /**
+     * Manually run the migrations for all apartments.
+     *
+     * Note: If a migration has not been ran via artisan it will not be applied until is has been!
```

Comment:
Reads a bit weird - "it will not be applied until is has been" - could clarify with (what I assume this means) "it will not be applied until artisan <some command ?> has been run".